mods/default/README.md

```diff
@@ -0,0 +1,387 @@
+- `default_coniferous_litter.png`
+- `default_coniferous_litter_side.png`
+- `default_grass.png`
+- `default_dry_dirt.png`
+
+BlockMen (CC BY-SA 3.0):
+- `default_aspen_leaves.png -- Derived from Sofar's texture`
+- `default_wood.png`
+- `default_clay_brick.png`
+- `default_iron_ingot.png`
+- `default_gold_ingot.png`
+- `default_tool_steelsword.png`
+- `default_diamond.png`
+- `default_tool_*.png`
+- `default_lava_source_animated.png`
+- `default_lava_flowing_animated.png`
+- `default_stick.png`
+- `default_chest_front.png`
+- `default_chest_lock.png`
+- `default_chest_side.png`
+- `default_chest_top.png`
+- `default_mineral_mese.png`
+- `default_meselamp.png`
+- `bubble.png`
+- `gui_formbg.png`
+- `gui_furnace_arrow_bg.png`
+- `gui_furnace_arrow_fg.png`
+- `gui_hb_bg.png`
+
+sofar (CC BY-SA 3.0):
+- `default_aspen_sapling`
+- `default_aspen_tree`
+- `default_aspen_tree_top, derived from default_pine_tree_top (by paramat)`
+- `default_aspen_wood, derived from default_pine_wood (by paramat)`
+- `default_chest_inside`
+
+sofar (CC0 1.0):
+- `default_gravel.png -- Derived from Gambit's PixelBOX texture pack light gravel`
+
+Neuromancer (CC BY-SA 3.0):
+- `default_cobble.png, based on texture by Brane praefect`
+- `default_mossycobble.png, based on texture by Brane praefect`
+- `default_furnace_*.png`
+
+Gambit (CC BY-SA 3.0):
+- `default_bronze_ingot.png`
+- `default_copper_ingot.png`
+- `default_copper_lump.png`
+- `default_iron_lump.png`
+- `default_gold_lump.png`
+- `default_clay_lump.png`
+- `default_coal.png`
+- `default_grass_*.png`
+- `default_paper.png`
+- `default_diamond_block.png`
+- `default_ladder_steel.png`
+- `default_sign_wall_wood.png`
+- `default_flint.png`
+- `default_snow.png`
+- `default_snow_side.png`
+- `default_snowball.png`
+- `default_key.png`
+- `default_key_skeleton.png`
+- `default_book.png`
+
+asl97 (CC BY-SA 3.0):
+- `default_ice.png`
+
+KevDoy (CC BY-SA 3.0):
+- `heart.png`
+
+Pithydon (CC BY-SA 3.0)
+- `default_coral_brown.png`
+- `default_coral_orange.png`
+- `default_coral_skeleton.png`
+
+Ferk (CC0 1.0):
+- `default_item_smoke.png`
+
+npx (CC BY-SA 3.0):
+- `default_rainforest_litter.png`
+- `default_rainforest_litter_side.png`
+
+kaeza (CC-BY-SA 3.0):
+- `default_desert_sandstone.png`
+- `default_desert_sandstone_brick.png`
+- `default_desert_sandstone_block.png`
+
+kilbith (CC BY-SA 3.0):
+- `default_steel_block.png`
+- `default_copper_block.png`
+- `default_bronze_block.png`
+- `default_gold_block.png`
+- `default_tin_block.png`
+- `default_mineral_tin.png`
+- `default_tin_ingot.png`
+- `default_tin_lump.png`
+
+tobyplowy (CC BY-SA 3.0):
+- `default_kelp.png`
+
+CloudyProton (CC BY-SA 3.0):
+- `default_book_written.png, based on default_book.png by Gambit`
+
+Mossmanikin (CC BY-SA 3.0):
+- `default_fern_*.png`
+
+random-geek (CC BY-SA 3.0):
+- `default_blueberries.png`
+- `default_blueberry_overlay.png`
+- `default_blueberry_bush_leaves.png, derived from default_bush_leaves (by paramat)`
```

Comment:
this should be
<pre>- `default_blueberry_bush_leaves.png`, derived from default_bush_leaves (by paramat)</pre>

same for other occurrences

Reply:
Fixed